hadoop-tools/hadoop-tools-dist/pom.xml

```diff
@@ -134,6 +134,24 @@
       <scope>compile</scope>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-msgraph</artifactId>
+      <scope>compile</scope>
+      <version>${project.version}</version>
+      <exclusions>
+        <!--
+          Avoiding conflicting dependency:
```

Comment:
If there's a conflict that needs to resolve within the project, else distributions will be in a mess & at least one module won't coexist